setup.py

```diff
@@ -75,19 +76,19 @@ def build_extension(self, ext):
     README = readme_file.read()
 
 setup(
-    name='jkq.qfr',
-    version='1.7.3',
-    author='Lukas Burgholzer',
-    author_email='lukas.burgholzer@jku.at',
-    description='QFR - A JKQ tool for Quantum Functionality Representation',
+    name='mqt.qusat',
+    version='0.0.1',
```

Comment:
this version should be the same as in the `CMakeLists.txt`, so I guess it should be `0.1.0`